tests/ignite/metrics/test_loss.py

```diff
@@ -75,7 +75,7 @@ def test_reset():
         loss.compute()
 
 
-def _test_distrib_compute_on_criterion(device):
+def _test_distrib_compute_on_criterion(device, tol=1e-6):
```

Comment:
Actually, here we would like to use reduced tolerance only for multinode case and keep default tolerance for other cases.

Reply:
Yes 've just made a fix for this, thanks for your comment